src/steps/fetch-account/converters.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  IntegrationInstance,
+  createIntegrationEntity,
+  Entity,
+} from '@jupiterone/integration-sdk-core';
+
+export function createAccountEntity(instance: IntegrationInstance): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: {},
+      assign: {
+        _key: `bugcrowd:account:${instance.id}`,
+        _type: 'bugcrowd_account',
+        _class: ['Account'],
+        name: instance.name,
+        displayName: instance.name,
+        description: instance.description,
+      },
+    },
+  });
+}
+
+export function createServiceEntity(instance: IntegrationInstance): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: {},
+      assign: {
+        _key: `bugcrowd:service:${instance.id}:bug-bounty`,
+        _type: 'bugcrowd_service',
+        _class: ['Service', 'Control'],
+        name: 'Bugcrowd Security Testing',
+        displayName: 'Bugcrowd Security Testing',
+        description: 'Crowd-sourced security testing',
+        category: ['security', 'software'],
+        function: 'bug-bounty',
```

Comment:
function can now be an array https://github.com/JupiterOne/data-model/blob/master/src/schemas/Control.json. The TODO can be resolved.